ui/tests/integration/components/pki-role-generate-test.js

```diff
@@ -69,4 +69,26 @@ module('Integration | Component | pki-role-generate', function (hooks) {
     assert.dom(SELECTORS.certificate).exists({ count: 1 }, 'shows certificate info row');
     assert.dom(SELECTORS.serialNumber).hasText('abcd-efgh-ijkl', 'shows serial number info row');
   });
+
+  test('it should display validation errors', async function (assert) {
+    assert.expect(2);
+
+    await render(
+      hbs`
+      <div class="has-top-margin-xxl">
+        <PkiRoleGenerate
+          @model={{this.model}}
+          @onSuccess={{this.onSuccess}}
+        />
+       </div>
+  `,
+      { owner: this.engine }
+    );
+    await click(SELECTORS.generateButton);
+
+    assert
```

Comment:
Sometimes we also check that the invalid input as the class `has-error-border` - you don't need to add it in this PR, but just pointing out as it can be easily missed when building forms that don't use the `FormFields` component (some of the less-common inputs in that component may also still be missing that class)

Actually, writing this is reminding me I need to add it to my PR! 😅 😂